### ISSUE TITLE
Incorrect type definition for 6th argument of add_settings_field()

### ISSUE THREAD

**Issue by owaisahmed5300** [humanmade/psalm-plugin-wordpress#62]
It looks like the type for the 6th argument of `add_settings_field()` in `psalm-plugin-wordpress` is too strict.

Currently, Psalm expects:

```php
array{class?: string, label_for?: string}
```

However, WordPress allows passing **any data** in this argument. The array is passed directly to the field render callback, which means developers can use custom keys as needed.

For example, instead of being limited to just `class` and `label_for`, the type should allow arbitrary keys:

```php
array{class?: string, label_for?: string, ...}
```

**Why this matters:**
Right now, Psalm reports false positives when extra keys are included in the `$args` array. Adjusting the type to include `...` would resolve this and better reflect WordPress behavior.

**Reference:**
[WordPress add\_settings\_field documentation](https://developer.wordpress.org/reference/functions/add_settings_field/) — the documentation explicitly lists `class` and `label_for`, but it does not restrict passing additional parameters.

**Comment by kkmuffme** [comment]
The stubs are from https://github.com/php-stubs/wordpress-stubs please report it there
